career/roles-in-tech/security.md

```diff
@@ -35,22 +35,22 @@ Apprentices will learn secure development basics, common pitfalls, and how to av
 
 _Participants will be able to learn:_
 
-- Risk Management
-- Cryptography & PKI
-- Identity and Access Management
-- How to implement secure n/w architecture and system design
-- Discover security vulnerabilities across an entire network
+- provide system, network and security knowledge
+- analyze and define security requirements
+- protect integrity, confidentiality and availability an information system
+- implement secure n/w architecture and system design
+- discover security vulnerabilities across an entire network
+- develop skills like - risk management, Identity and Access Management, Programming Skills, Communication Skills
 
 ### Specific Things to Learn
 
 _The basic required skills as a Security Engineer are_
```

Comment:
This should be a different section under Lesson.  But "The basic required skills as a Security Engineer" could be one of the Specific Things to Learn.  This should just be a list/summary of what's ahead - no one should actually learn anything in this section.